prep-javascript-objects/main.js

```diff
@@ -0,0 +1,19 @@
+const person = {
+  firstName: 'Bug',
+  lastName: 'Smith',
+  hobby: 'golfing',
+  job: 'unemployed',
+  previousJob: 'Techstack Admin',
```

Comment:
the job property should be added to the object using dot notation. 
the previous job property should be added to the object using bracket notation.